pandas/tests/indexes/multi/test_sorting.py

```diff
@@ -105,7 +105,11 @@ def test_unsortedindex():
     expected = df.iloc[0]
     tm.assert_series_equal(result, expected)
 
-    with pytest.raises(UnsortedIndexError):
+    msg = (
+        r"MultiIndex slicing requires the index to be lexsorted: "
```

Comment:
```suggestion
        "MultiIndex slicing requires the index to be lexsorted: "
```

Reply:
A single line is split into multiple lines, to adhere to the formatting rules. If I were to do this, one of the lines will be `""` and other will be `r""`.